Makefile

```diff
@@ -85,7 +85,7 @@ coverage:
 		rm -f coverage-tmp.txt;  \
 		$(GO) test -coverprofile=coverage-tmp.txt $$p ; \
 		if [ -f coverage-tmp.txt ]; then \
-			cat coverage-tmp.txt |grep -v 'mode:' >> coverage.txt; \
+			cat coverage-tmp.txt |  grep -v 'mode:' >> coverage.txt || /bin/true; \
```

Comment:
I don't get it, why has this started happening now? You didn't change any of the code, so the coverage should be the same as before. Which files are the problematic ones?

Reply:
It has not started happening now. Checkout master and run the command. This was already broken. I just fixed it to get a green pipe. 

Do you want a separate PR for it?

Reply:
No, that's ok. Sorry, I missed the fact that it was already broken. Looks good then!